DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -734,6 +874,36 @@ struct qaMatchEff {
     histos.add("MC/PID/zDCA_tpcits_pr", "DCA along z - protons TPC+ITS tag;dca [cm]", kTH1D, {{200, -20.0, 20.0}}, true);
     histos.add("MC/PID/xyDCA_tpcits_pr", "DCA in x-y plane - protons TPC+ITS tag;dca [cm]", kTH1D, {{200, -20.0, 20.0}}, true);
 
+    //
+    // pions only
+    // all
+
+    //
+    histos.add("MC/TPCclust/tpcNClsFound_pi", "Number of TPC found clusters - pions", kTH1F, {{161, -0.5, 160.5}}, true);
```

Comment:
what about the cases `tpcNClsFindable_pi`, `tpcCrossedRows_pi` and `tpcsFindableMinusCrossedRows_pi`?

Reply:
this is for "all" pions, without any selection but the pid, to keep the task from diverging in size tried to put this one as a reference. It is true that ~10 histos more will not make a real difference... Real point is that should review this task :)